pyvista/core/filters.py

```diff
@@ -1195,6 +1199,19 @@ def glyph(dataset, orient=True, scale=True, factor=1.0, geom=None,
         progress_bar : bool, optional
             Display a progress bar to indicate progress.
 
+        Returns
+        -------
+        glyphs : pyvista.PolyData
+            Glyphs at either the cell centers or points.
+
+        Examples
+        --------
+        Create arrow glyphs oriented by vectors and scaled by scalars.
+
+        >>> import pyvista as pv
+        >>> from pyvista import examples
+        >>> mesh = examples.download_carotid().threshold(145, scalars="scalars")
```

Comment:
Does this need to be skipped in DOCTEST?  I see examples of both in the code base, and I'm not sure what the dividing line between skipping and not skipping is.

Reply:
> I'm not sure what the dividing line between skipping and not skipping is.

We should make that clear.  Right now, I think (and this may not still be the case), but we have to skip anything that plots or downloads.  Plotting is an issue as we might not have the xserver up, and downloading is an issue on Mac on Azure.